checks.md

```diff
@@ -776,3 +776,20 @@ kubectl taint node <node-name> <taint-key>-
 ```
 
 Note the trailing `-` on the key; this causes `kubectl` to delete the label or taint.
+
+## Images hosted on docker.pkg.github.com
+
+- Name: `docker-pkg-github-com-registry`
+- Groups: `containerd`, `doks`
+
+`containerd` cannot pull container images from `docker.pkg.github.com` due to a [protocol mismatch](https://github.com/containerd/containerd/issues/3291#issuecomment-683700425). As `docker.pkg.github.com` is GitHub's old package [they recommend migrating to `ghcr.io`](https://docs.github.com/en/packages/guides/migrating-to-github-container-registry-for-docker-images#domain-changes).
```

Comment:
Nit: suggest "hostname" instead of "package" here.

Reply:
Changed it to `package registry`. It isn't a simple hostname change. I've images on both, and they are different product within GitHub with different possibilities. `docker.pkg.github.com/OWNER/REPO/IMAGE:tag` vs `ghcr.io/OWNER/IMAGE:tag` is an externally visible difference.

Reply:
Ah, yes, good point - the new wording is good.